### PR TITLE
Dev to Stage

### PR DESCRIPTION
**Pull request by glespinosa**
- Created static layout components (https://docs.google.com/spreadsheets/d/1AYE9XnRVq8XRVOgd-G8g9yLqhDX4FuKkvdRU4EkQuJA/edit#gid=0)
- Component selector layouts
- resolved issue in navigation in mindshare not showing properly when user is logged in

### CODE REVIEW

File: src/revamp/ui/LightHero/index.js [hunk 1]
```diff
@@ -14,16 +15,14 @@ const schemaPic =
...
-
 const LightHero = ({
   title = 'Unify your team with',
-  header = 'Maximize Content Marketing Performance',
-  subtitle = 'Drive business growth with a visual CMS to create, deliver, measure, and optimize your content marketing at scale.',
-  primaryBtn = 'Start Now',
-  primaryBtnLink = '#',
-  secondaryBtn = 'Contact Sales',
-  secondaryBtnLink = '#',
+  header = 'The future proof CMS for enterprise of any size',
```
Comment (ardeay) [inline src/revamp/ui/LightHero/index.js]: This needs to be reverted to the original text

File: src/lib/ZestyView.js [hunk 2]
```diff
@@ -48,6 +51,8 @@ export function ZestyView(props) {
    * npm run sync must be run! otherwise it default back to auto layout component
    */
 
+  console.log(useAutoLayoutCheck());
```
Comment (ardeay) [inline src/lib/ZestyView.js]: remove console log

**Review by ardeay — CHANGES_REQUESTED** [review]

File: src/lib/ZestyView.js (hunk 2 above)
Comment (glespinosa) [reply src/lib/ZestyView.js]: done

File: src/revamp/ui/LightHero/index.js (hunk 1 above)
Comment (glespinosa) [reply src/revamp/ui/LightHero/index.js]: done

File: src/revamp/ui/LightHero/index.js [hunk 3]
```diff
@@ -14,16 +15,14 @@ const schemaPic =
...
-
 const LightHero = ({
   title = 'Unify your team with',
   header = 'Maximize Content Marketing Performance',
-  subtitle = 'Drive business growth with a visual CMS to create, deliver, measure, and optimize your content marketing at scale.',
-  primaryBtn = 'Start Now',
-  primaryBtnLink = '#',
-  secondaryBtn = 'Contact Sales',
-  secondaryBtnLink = '#',
+  subtitle = 'Create, distribute, and optimize content at SCALE with fewer resources in less time with the Zesty Hybrid Headless CMS. ',
```
Comment (ardeay) [inline src/revamp/ui/LightHero/index.js]: This also needs to be reverted

Comment (ardeay) [reply src/revamp/ui/LightHero/index.js]: This whole file needs to be reverted, what has been changed is older language.

**Review by ardeay — CHANGES_REQUESTED** [review]
Revert all language from that file. Update from latest stage

File: src/revamp/ui/LightHero/index.js (hunk 3 above)
Comment (glespinosa) [reply src/revamp/ui/LightHero/index.js]: done

**Review by ardeay — APPROVED** [review]

**Pull request merged**